carson-jardine/Crate
# Feature/user update

#### What’s this PR do? 
This PR:
1. populates the `Profile` profile page after the `EditProfile` form is submitted.
2. Builds form validation in `EditProfile` with required fields.
3. Builds out design for eventual `OrderHistory` component
4. Adds a date input to each subscription card in `Subscription`.

#### Where should the reviewer start?
Review `EditProfile` and `Profile`

#### How should this be manually tested?
1. Fill out the `EditProfile` form and submit.
2. Check to make sure new user info appears in the `Profile` page.
3. There are required fields. Try submitting the form without filling out those inputs and submit.
4. Check to make sure there is an error message and you do not redirect back to the `Profile` page.

#### Any background context you want to provide? 
There are a few backend rebases in this branch.

#### What are the relevant tickets? 
#33 
#35
